src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedFieldLikeEventAccessorSymbol.cs

```diff
@@ -11,35 +11,20 @@
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     /// <summary>
-    /// Event accessor that has been synthesized for a field-like event declared in source.
+    /// Event accessor that has been synthesized for a field-like event declared in source,
+    /// or for an event re-abstraction in an interface.
     /// </summary>
     /// <remarks>
-    /// Associated with <see cref="SourceFieldLikeEventSymbol"/>.
+    /// Associated with <see cref="SourceFieldLikeEventSymbol"/> and <see cref="SourceCustomEventSymbol"/>.
     /// </remarks>
-    internal sealed class SynthesizedFieldLikeEventAccessorSymbol : SourceEventAccessorSymbol
+    internal sealed class SynthesizedEventAccessorSymbol : SourceEventAccessorSymbol
     {
         // Since we don't have a syntax reference, we'll have to use another object for locking.
         private readonly object _methodChecksLockObject = new object();
 
-        private readonly string _name;
-
-        internal SynthesizedFieldLikeEventAccessorSymbol(SourceFieldLikeEventSymbol @event, bool isAdder)
-            : base(@event, null, @event.Locations)
-        {
-            this.MakeFlags(
-                isAdder ? MethodKind.EventAdd : MethodKind.EventRemove,
-                @event.Modifiers,
-                returnsVoid: false, // until we learn otherwise (in LazyMethodChecks).
-                isExtensionMethod: false,
-                isMetadataVirtualIgnoringModifiers: false);
-
-            _name = GetOverriddenAccessorName(@event, isAdder) ??
-                SourceEventSymbol.GetAccessorName(@event.Name, isAdder);
-        }
-
-        public override string Name
```

Comment:
>Name [](start = 31, length = 4)

This removal seems surprising, as it means the `Name` will be empty (implementation from `Symbol`, I think). Wouldn't that at least affect some public APIs (`ISymbol.Name` now gives a different answer)?